sysutils/beats8/src/opnsense/mvc/app/models/OPNsense/Beats8/Filebeat.xml

```diff
@@ -0,0 +1,55 @@
+<model>
+    <mount>//OPNsense/filebeat</mount>
+    <description>
+        Send logs to elasticsearch
+    </description>
+    <items>
+        <enabled type="BooleanField">
+            <Default>0</Default>
+            <Required>Y</Required>
+        </enabled>
+        <modules>
+            <enabled type="OptionField">
+                <OptionValues>
+                    <suricata>Suricata (Intrusion Detection)</suricata>
+                </OptionValues>
+                <Multiple>Y</Multiple>
+            </enabled>
+        </modules>
+        <inputs>
+            <enabled type="OptionField">
+                <Default>audit</Default>
```

Comment:
missing required?

Reply:
@0xThiebaut in these cases I usually prefer to extend the validation using `performValidation()`, for example dnsmasq has this.

https://github.com/opnsense/core/blob/7c37b352ab4d86a00e0e21a83835b14c1d5d6f41/src/opnsense/mvc/app/models/OPNsense/Dnsmasq/Dnsmasq.php#L52

call the parent:

https://github.com/opnsense/core/blob/7c37b352ab4d86a00e0e21a83835b14c1d5d6f41/src/opnsense/mvc/app/models/OPNsense/Dnsmasq/Dnsmasq.php#L56

make sure to return at the end:

https://github.com/opnsense/core/blob/7c37b352ab4d86a00e0e21a83835b14c1d5d6f41/src/opnsense/mvc/app/models/OPNsense/Dnsmasq/Dnsmasq.php#L187

and in the middle you can validate any rocket science you might need ;)

Reply:
Best add a `hint` in the form and remove the default, this will likely trip our lint check.

Reply:
Removed the default but did not add a hint as it makes the distinction between hinted no selection and selected hint visually tricky. The default doesn't matter given validation is now in place.